src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContextOfT.cs

```diff
@@ -56,6 +56,8 @@ public override async Task<bool> ProcessRequestAsync()
                 }
             }
 
+            _streams.Stop();
```

Comment:
Can `_streams.Stop()` go into the finally block below where we are awaiting for the read and write tasks?